### DISCUSSION TITLE
Version 1.8.0 released, but missing on Nuget.org?

### DISCUSSION THREAD

**Discussion by julealgon** [open-telemetry/opentelemetry-dotnet#5499] [Q&A] [answered]
I'm genuinely confused by this. I saw the release notes earlier today about the v1.8.0 release:
- https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.8.0

Then I went to update the packages just now on our project and I wasn't finding the version in the NuGet package manager UI. I closed and reopened it a couple times before double checking in www.nuget.org, only to find that the version hadn't been pushed at all aparently.

- https://www.nuget.org/packages/OpenTelemetry.Exporter.OpenTelemetryProtocol

![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/461579/f0b84eb5-9ec1-48e9-915d-1623cbe6573b)

Am I missing something obvious on this one? Is there some sort of expected delay between the releases here on github and the package actually being pushed into nuget.org? Or was this some sort of mistake?

**Comment by CodeBlanch** [comment, marked as answer]
Sorry about that @julealgon! They will be pushed before the end of the day. What's going on here is I needed to create the release on GitHub in order to kick off the job to create the packages. It pushes to MyGet: https://github.com/open-telemetry/opentelemetry-dotnet/actions/runs/8527337240/job/23358563266#step:8:1

I took those packages and they are currently running through some scanning tools to make sure they are safe.

Once that scan is done I'll push them to NuGet. This is a new thing we're trying and at the moment it is a manual process. Hopefully we'll be able to automate it and improve things in the future!

**Reply by julealgon** [reply]
I see, thanks for the explanation @CodeBlanch !
